### ISSUE TITLE
Slides for Bob - due June 26

### ISSUE THREAD

**Issue by kristinashu** [MozillaFoundation/Design#597]
Update diagrams for Bob's presentation this Friday to some funders. 

https://docs.google.com/presentation/d/1us-sjZ3dpgeVTEeJLY8HCqU76ZxnmJTqSl8dlpne2b4/edit?ts=5ef21d71#slide=id.g819be3fb31_0_40

**Comment by kristinashu** [comment]
@sabrinang thank you for offering to do this. I believe Sarah W will reach out to you with more info.

**Comment by sabrinang** [comment]
Sarah reached on email indicating that diagrams on slide 1/7 needed design help. Bob hasn't replied to provide input on what design elements were needed yet.

So far I iterated on diagrams from slides 1 and 7 in the doc beneath the corresponding slides (on 3 and 9): https://docs.google.com/presentation/d/1us-sjZ3dpgeVTEeJLY8HCqU76ZxnmJTqSl8dlpne2b4/edit?usp=sharing 

I emailed back for feedback as I don’t have much context of the content and didn't hear what specific design elements needed. Asked to chat tomorrow (slack/zoom) to give me a better understanding of the diagrams so I can have it ready for Friday.

Some things I mentioned for clarification:
**Design Futures Lab diagram**
- Is the “public goods…” and “specific projects…” feeding into the lab or is it just describing the sections
- I wasn’t quite sure on the relationship of dotted line box (if it pertains to the two relationship arrows only or the whole lab)
- Phrasing of “network support…” (can this be short to match other diagram elements)

**Strategy Governance Mechanisms diagram**
- I don’t really understand the arrow relationship between the mechanism and the network (is it one way? is it for the system or just between the lab team and network?)
